### DISCUSSION TITLE
how to change the style of a CTKwidget

### DISCUSSION THREAD

**Discussion by zorrolaotse** [TomSchimansky/CustomTkinter#2367] [Q&A] [answered]
HI,
is it possible to change the style of a CTKwidget (arrow size of a CTKComboBox for example) ?
I've tried to use the tkk.style but without result.
More generaly how to access to the tkinter components of of CTKwidget ?

Thanks

**Comment by avalon60** [comment]
Have you checked out CTkComboBox.configure api in the docs?
https://customtkinter.tomschimansky.com/documentation/widgets/combobox

**Reply by zorrolaotse** [reply]
Yes, configure can change the font of the text or the items of the dropdown but not button on the right of the combobox.

**Comment by rigvedmaanas** [comment, marked as answer]
Hi @zorrolaotse,

It is not possible to configure the style of the dropdown arrow. The dropdown arrow is created by the draw engine (canvas). 

```python
requires_recoloring_2 = self.draw_engine.draw_dropdown_arrow(self._apply_widget_scaling(self._current_width - (self._current_height / 2)),  self._apply_widget_scaling(self._current_height / 2), self._apply_widget_scaling(self._current_height / 3))
```

The above code in the `ctk_combobox.py` draws the dropdown arrow.

The dropdown arrow color is linked to the font color, so changing the font color will also change the arrow color. However, you can independently change the color of the dropdown. But it's not recommended since it might change again back to the original color when you configure the button.

Example Code
```python
combo = CTkComboBox(root, values=["Value 1", "Value 2", "Value 3"])
combo.pack(expand=True)
combo._canvas.itemconfig("dropdown_arrow", fill="yellow")
```
Result
![result](https://github.com/TomSchimansky/CustomTkinter/assets/77579661/973f6712-80e3-434c-ab61-6ff3fb9ae42b)


I hope this helps

**Reply by zorrolaotse** [reply]
Thanks for your answer
